src/port.js

```diff
@@ -0,0 +1,18 @@
+function Port(name) {
+    this.currentPort = 'Dover';
+    this.ships = [];
+};
+
+  Port.prototype.addShip = function addShip(ship) {
+    this.ships.push(ship);
+  };
+
+  Port.prototype.removeShip = function removeShip(ship) {
+    this.ships.pop(ship);
+  };
+ 
+  
+
+module.exports = Port;
+
+
```

Comment:
Formatting again here. As said above you dont need more than one consecutive blank line in your code, also you have varying levels of indentation, sometimes 2 sometimes 4. Remember that you only need to indent when you go inside a function, so in this case you can move the functions on lines 6 and 10 back a level to the same as the opening function on line 1:

```
function Port(name) {
  this.currentPort = 'Dover';
  this.ships = [];
};

Port.prototype.addShip = function addShip(ship) {
  this.ships.push(ship);
};

Port.prototype.removeShip = function removeShip(ship) {
  this.ships.pop(ship);
};

module.exports = Port;

```